documentation/book/assembly-using-kafka-connect-with-plugins.adoc

```diff
@@ -6,16 +6,13 @@
 // This is necessary for including assemblies in assemblies.
 // See also the complementary step on the last line of this file.
 
-[id='using-kafka-connect-with-plugins-{context}']
-= Using Kafka Connect with plugins
+[id='using-kafka-connect-with-plug-ins-{context}']
+= Extending Kafka Connect with plug-ins
 
-{ProductName} container images for Kafka Connect contain, by default, only the `FileStreamSinkConnector` and `FileStreamSourceConnector` connectors which are part of Apache Kafka.
+The {ProductName} container images for Kafka Connect include the two built-in connectors: `FileStreamSourceConnector` and `FileStreamSinkConnector`. You can add your own connectors by using one of the following methods:
```

Comment:
two shouldn't be spelled out.

Reply:
Wait ... yes it should. I always screw these up.  It's numbers over nine that are not spelled out. ignore me @laidan6000

Reply:
Ha ha, OK ;)